back-office/wishlist.md

```diff
@@ -0,0 +1,337 @@
+# Wishlist native module functional specifications
+
+## Feature 1: Add to favorites 
+
+### US 1: Add a product to a favorites list
+##### As a logged-in user, I can add a product to my favorites from the product sheet or any other product listing page with product blocks in a favorites list
+Several cases: 
+- Connected users 
+- Non logged-in users 
+- Products with combinations & customizations
+
+Functional rules: 
+What type of users can add products to their favorites?  
+Only logged-in users can add a product to their favorites - provided they click on the heart on the product sheet or a product block from a navigation page
+
+![wishlist screen 1](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-1.png)
+
+
+Which products can be added to favorites? 
+All products can be added as favorites:  
+- Products in and out of stock  
+- Standard products
+- Products with combinations
+- product packs
+- virtual products
+- customizable products (text or file)
+
+What product information do I need to retrieve for favorites?
+All product information:  
+- Price excl. VAT or incl. VAT + special offers,  
+- Photos,  
+- Quantity (selected by the customer),  
+- Combinations,
+- Customization (if the product can be customised)
+
+Product information (stocks, prices, etc.) is variable and must be updated each time the Favorites page is loaded.
+
+Rules for quantities when adding to favorites :
+- If the user has chosen a quantity > 1 the product added in favorites has the same quantity as the one on the product sheet
+- By default, the product is added to the favorites by unit, unless the product can only be purchased with a minimum of X units, in which case, the product is added to the favorites with its minimum quantity
+- If the product is out of stock without Allow order option (stock = 0 ou stock < 0): it is saved in favorites with 1 product quantity but the product block is greyed out and the visitor cannot add the product to the basket  
+
+Products with combinations:
+- When a user adds a product from a list page (where they cannot select a combination), the product added to the wishlist is the "default" product combination. That is, the product is added with its default combination and quantity.  
+- If the user selects a combination in the product sheet, then this combination is added, not the "default" product combination
+
+Customizable products : 
+No matter if the user has chosen his customisation or not, when he adds his customisable product to his favorites, it will be saved without his customisation. 
```

Comment:
```suggestion
No matter if the user has filled his customization or not, when he adds his customizable product to his favorites, it will be saved without his customization. 
```